### ISSUE TITLE
Log Viewer - message in expandable message cell can overflow the message frame

### ISSUE THREAD

**Issue by karel-rehor** [influxdata/chronograf#4610]
Build **Nightly Build**
Build time **201810172304**
Browser **Chrome 69**
```
$ chronograf --version
2018/10/18 13:41:27 Chronograf 201810172304~nightly (git: db9b32e775530521bfd86334a91d2966ecfa0124)
```

*Use Case*

While preparing Selenium tests for Chronograf logs, I've captured a message from chronograf that fails to stay wrapped inside the expanded message container.  See screenshot below. 

*Expected Behavior*

That messages should be wrapped inside their containing elements and should not overflow into other parts of the layout. 

*Screenshot*
![logmessageoverflow01](https://user-images.githubusercontent.com/25881301/47152104-0565dd00-d2dc-11e8-9c06-e49d3efe863c.png)

**Comment by karel-rehor** [comment]
How to reproduce on Linux. 

1. Use logger to send an extremely long message with url encodings to syslog:
e.g.
```
$ logger "AHOJ20%AHOJ20%AHOJ20%AHOJ20%AHOJ20%AHOJ20%AHOJ20%AHOJ20%AHOJ20%AHOJ20%AHOJ20%AHOJ20%AHOJ20%AHOJ20%AHOJ20%AHOJ20%AHOJ20%AHOJ20%AHOJ20%AHOJ20%AHOJ20%AHOJ20%" 
```
2. Wait for the message to be processed from syslog to telegraf to influx
3. When the message arrives switch the view mode to _wrap_.  The message wraps successfully,
4. Switch the view mode back to _truncate_ and click on the message to activate the message _cell viewer_ 

The message overflows the viewer borders. 

*Additional Screenshots*
![messageoverflow02](https://user-images.githubusercontent.com/25881301/47156183-d43eda00-d2e6-11e8-9ed9-bc681b209f8c.png)
![messageoverflow03](https://user-images.githubusercontent.com/25881301/47156184-d4d77080-d2e6-11e8-82f2-e9d62cfe6ed5.png)

**Comment by OfTheDelmer** [comment]
Closed by #4621
